compiler/src/dotty/tools/dotc/reporting/diagnostic/ErrorMessageID.java

```diff
@@ -136,6 +136,9 @@
     ValueClassParameterMayNotBeCallByNameID,
     NotAnExtractorID,
     MemberWithSameNameAsStaticID,
+    TraitCompanionWithMutableStaticID,
+    LazyStaticFieldID,
+    StaticOverridingNonStaticMembersID,
     PureExpressionInStatementPositionID
```

Comment:
All new error message IDs must go at the end

Reply:
even when they are related in context ?

Reply:
Yes, it is to have a stable enumeration id